lib/top/mdx_top.ml

```diff
@@ -779,6 +779,9 @@ let rec save_summary acc s =
         else acc
       in
       save_summary acc summary
+#if OCAML_MAJOR >= 4 && OCAML_MINOR >= 8
+  | Env_module (_, _, Mp_absent, _)
+#endif
   | Env_empty -> acc
```

Comment:
I'm not sure I understand what the purpose of this `save_summary` function is, but here you're stopping the recursion at the first absent module that you encounter, and that seems like a mistake.

Reply:
From what I remember, I think this function is used to be able to have several environments at the same time.
But indeed the recursion is broken by my changes, I'll get that solved